autoload/goshiteki.vim

```diff
@@ -18,6 +18,15 @@ function! g:goshiteki#start_review() abort
   let l:name = l:owner_and_name[-1]
 
   let l:pr = split(system([s:script_dir . 'pr-id.sh', trim(l:owner), trim(l:name), trim(l:current_branch)]), "\n")
+
+  if len(l:pr) == 0
+    echoerr 'A pull request is not found.'
+    return
+  elseif len(l:pr) == 1
+    echoerr l:pr[0]
+    return
```

Comment:
When `owner` or `name` is invalid, the message below is displayed so it is displayed as it is as error message.
```
gh: Could not resolve to a Repository with the name ':owner/:name'.
```

Reply:
The trouble is that the `v:shell_error` is always `0` because of the fixed jq query. If I don't fix jq, the error existence can be detected by `v:shell_error`. However, the cause of the error cannot be determined  by `owner` or `name` or by `current_branch`.

Reply:
Just use `set -o pipefail`. Don't mess around with that!